composer.json

```diff
@@ -15,6 +15,11 @@
             "name": "Alessandro Schmitz",
             "role": "Developer",
             "homepage": "https://www.interlutions.de/"
+        },
```

Comment:
Please do not mention you as an author.
I will mention you and your company in the CHANGELOG.md